src/DssSpell.sol

```diff
@@ -49,82 +127,196 @@ contract DssSpellAction is DssAction {
+            // Set Proportional Kick Incentive (chip) to 0
+            DssExecLib.setKeeperIncentivePercent(_ilk, 0);
+            // Set Flat Kick Incentive (tip) to 0
+            DssExecLib.setKeeperIncentiveFlatRate(_ilk, 0);
+        }
+        {
+            bytes32 _ilk  = bytes32("PAXUSD-A");
+            address _clip = DssExecLib.getChangelogAddress("MCD_CLIP_PAXUSD_A");
+            //
+            // Enable liquidations for PAXUSD-A
+            // Note: ClipperMom cannot circuit-break on a DS-Value but we're adding
+            //       the rely for consistency with other collaterals and in case the PIP
+            //       changes to an OSM.
+            DssExecLib.authorize(_clip, DssExecLib.clipperMom());
+            DssExecLib.setValue(_clip, "stopped", 0);
+            // Use Abacus/LinearDecrease
+            DssExecLib.setContract(_clip, "calc", MCD_CLIP_CALC_PAXUSD_A);
+            // Set Liquidation Penalty to 0
+            DssExecLib.setIlkLiquidationPenalty(_ilk, 0);
+            // Set Auction Price Multiplier (buf) to 1
+            DssExecLib.setStartingPriceMultiplicativeFactor(_ilk, 100_00);
+            // Set Local Liquidation Limit (ilk.hole) to 3m DAI
+            DssExecLib.setIlkMaxLiquidationAmount(_ilk, 3_000_000);
+            // Set tau for Abacus/LinearDecrease to 4,320,000 second
+            DssExecLib.setLinearDecrease(MCD_CLIP_CALC_PAXUSD_A, 4_320_000);
+            // Set Max Auction Duration (tail) to 43,200 seconds
+            DssExecLib.setAuctionTimeBeforeReset(_ilk, 43_200);
+            // Set Max Auction Drawdown / Permitted Drop (cusp) to 0.99
+            DssExecLib.setAuctionPermittedDrop(_ilk, 99_00);
+            // Set Proportional Kick Incentive (chip) to 0
+            DssExecLib.setKeeperIncentivePercent(_ilk, 0);
+            // Set Flat Kick Incentive (tip) to 0
+            DssExecLib.setKeeperIncentiveFlatRate(_ilk, 0);
+        }
+
+        // ----------------- Whitelist Light Feed on Oracle for rETH -----------------
+        // https://forum.makerdao.com/t/whitelist-light-feed-for-reth-oracle/18908
+        require(OSMLike(DssExecLib.getChangelogAddress("PIP_RETH")).src() == RETH_ORACLE, "Bad oracle address");
+        address[] memory lightFeeds = new address[](1);
+        lightFeeds[0] = RETH_LIGHTFEED;
+        OracleLiftLike(RETH_ORACLE).lift(lightFeeds);
+
+        // ------------------ Setup new Starknet Governance Relay -----------------
+        // Forum: https://forum.makerdao.com/t/starknet-changes-for-executive-spell-on-the-week-of-2022-11-29/18818
+        // Relay l2 part of the spell
+        // L2 Spell: https://voyager.online/contract/0x013c117c7bdb9dbbb45813fd6de8e301bbceed2cfad7c4c589cafa4478104672#code
+        StarknetGovRelayLike(STARKNET_GOV_RELAY).relay(L2_GOV_RELAY_SPELL);
+
+        // ----------------- MKR Transfer -----------------
+        TokenLike(DssExecLib.mkr()).transfer(DUX_WALLET, 180.6 ether);
```

Comment:
Previous pattern is to use `GemLike` for MKR when using the structure you use here.

For example:
```GemLike(DssExecLib.mkr()).transfer(SOME_WALLET, AMT * UNIT);```

Reply:
This is fine either way imo.